integration/analyzer_peliasAdmin.js

```diff
@@ -103,4 +103,4 @@ function simpleTokens( tokens ){
   return tokens.map( function( t ){
     return t.token;
   });
-}
\ No newline at end of file
+}
```

Comment:
What? You've finally bowed down to the true UNIX ways of newline chars at the end of the last line of a file? there is hope